copier/main.py

```diff
@@ -1105,8 +1105,15 @@ def _apply_update(self) -> None:  # noqa: C901
                         )
                         # Remove rejection witness
                         Path(f"{fname}.rej").unlink()
-                        # Store file name for marking it as unmerged after the loop
-                        conflicted.append(fname)
+                        # The 3-way merge might have resolved conflicts automatically,
+                        # so we need to check if the file contains conflict markers
+                        # before storing the file name for marking it as unmerged after the loop.
+                        with open(fname) as conflicts_candidate:
+                            if any(
+                                line.rstrip() in ("<<<<<<< before updating", ">>>>>>> after updating")
```

Comment:
issue: a little nitpick. In general, using sets is better for `in` checks:

```suggestion
                                line.rstrip() in {"<<<<<<< before updating", ">>>>>>> after updating"}
```

Reply:
True! Sometimes though I wonder if hashing each line is more or less performant than just comparing a few characters. Anyway, lets just follow the general rule and use a set :+1: :smile: